src/app/mork/edx/mixins.py

```diff
@@ -64,3 +70,30 @@ def get_inactive_users(
             .limit(limit)
         )
         return session.scalars(query).unique().all()
+
+    @classmethod
+    def delete_user(cls, session: Session, usermail: str, username: str) -> int:
+        """Deletes a user entry based on the provided email and username.
+
+        Parameters:
+        session (Session): SQLAlchemy session object.
+        usermail (str): The email of the user to delete.
+        username (str): The username of the user to delete.
+
+        Returns:
+        int: 1 if deletion was successful, 0 if user not found.
+        """
+        try:
+            result = (
+                session.query(cls)
+                .filter(cls.email == usermail, cls.username == username)
+                .delete(synchronize_session=False)
```

Comment:
We could use the newer `delete` API instead of the legacy Query API (doc [here](https://docs.sqlalchemy.org/en/20/orm/queryguide/dml.html#orm-update-and-delete-with-custom-where-criteria))